CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing guide
+
+We're always looking for new contributors and are happy to help you get started! Some ideas:
+
+- Add a test for a function.
+- Improve a function's documentation.
+- Work on any issue tagged [good first issue](https://github.com/remeda/remeda/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+- Work on any issue tagged [help wanted](https://github.com/remeda/remeda/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
+
+## Getting started
+
+Requirements:
+
+- Node.js `>=18.20.0`
+- npm `>=10.7.0`
+
+Install dependencies:
+
+```bash
+git clone git@github.com:remeda/remeda.git
+cd remeda
+npm install
+```
+
+Run a test, and watch for changes:
+
+```bash
+npm test -i src/file.test.ts
+```
+
+We have a pre-commit script that will format and lint the project.
+
+## Documentation
+
+Help us make the documentation better! See [`docs/README.md`](https://github.com/remeda/remeda/blob/main/docs/README.md).
```

Comment:
Maybe add something about the fact that for most use-cases the documentation is generated from the JSDocs so effectively most documentation PRs wouldn't actually be part of the docs project.